### ISSUE TITLE
(node:6332) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated

### ISSUE THREAD

**Issue by masx200** [spdy-http2/node-spdy#366]
node -v
v13.13.0

(node:6332) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated

https://github.com/spdy-http2/node-spdy/blob/661b14a8db40a76a9a4842fc12ca8908f989cfd9/lib/spdy/response.js#L18

```js
  if (this._headers) {
```

**Comment by FlorentinTh** [comment]
I just made a pull request to fix it : [#367](https://github.com/spdy-http2/node-spdy/pull/367)

[EDIT] but obviously the CI fails on node 6 since the feature was introduced in version 7. If maintainers need to continue supporting node 6 maybe we will not have a release with the merge coming soon. 

[EDIT-2] I was wrong in my first EDIT I found a way to support older versions of node.

**Comment by masx200** [comment]
Support it. Great. Well done.

**Comment by MatsG23** [comment]
I am also getting 
(node:12775) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
(Use `node --trace-deprecation ...` to show where the warning was created)

I'm using Node.js 14.1.0 and spdy 4.0.1

**Comment by FlorentinTh** [comment]
@MatsG23 yeah theoretically my pull request will fix all possible warnings however I cannot figure out how to make tests to work with node 12.x and 14.x. Moreover, the project seems to be left off since these newer versions of node include http2 and it is plan in the express 5 roadmap to support http2 which is currently not supported in v4.x

**Comment by masx200** [comment]
@indutny @anandsuresh @eee-c @anandsuresh @daviddias

**Comment by krishnaTORQUE** [comment]
I am also getting the same warning 
```
(node:57156) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
```

**Comment by therealgilles** [comment]
Any chance to get this fixed soon?

**Comment by euberdeveloper** [comment]
This deprecation warning was introduced I think with NodeJS 12 and I see that the npm module is last updated one year ago. Is this module dead?

**Comment by sido420** [comment]
Any updates on this? These warnings are long and annoying
